src/components/dialogs/export-network/flat-parameters-input.tsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { ExportFormatProperties } from '../../../services/study';
+import { useCallback, useMemo, useState } from 'react';
+import { useController, useFormContext } from 'react-hook-form';
+import { FlatParameters, Parameter } from '@gridsuite/commons-ui';
+import { IGNORED_PARAMS } from '../root-network/ignored-params';
+import { Collapse, IconButton, Stack, Typography } from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+import ExpandLessIcon from '@mui/icons-material/ExpandLess';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+
+export interface FlatParametersInputProps {
+    name: string;
+    fileFormatName: string;
+    parameters?: Record<string, ExportFormatProperties>;
+}
+
+export function FlatParametersInput({ name, parameters, fileFormatName }: Readonly<FlatParametersInputProps>) {
+    const [unfolded, setUnfolded] = useState(false);
+
+    const {
+        field: { onChange, value },
+    } = useController({ name });
+
+    const { getValues } = useFormContext();
+    const exportFormat: string = getValues(fileFormatName);
```

Comment:
I have another solution to make this component less depend on exportFormat, see
https://github.com/gridsuite/gridstudy-app/pull/3548